docs/book/src/reference/admission-webhook.md

```diff
@@ -99,5 +99,3 @@ spec:
 While certain edge scenarios might allow a mutating webhook to seamlessly modify the status, treading this path isn't a 
 universally acclaimed or recommended strategy. Entrusting the controller logic with status updates remains the 
 most advocated approach.
-
-[k8s-doc-admission-webhooks]: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks
```

Comment:
We should use an alias as it is now so that it makes it easier us keep things maintained (allows re-usage, easily identify where we need to update etc)

So, can you please share with us what is the motivation for this PR?
is the link wrong?

Reply:
The rendered page on the website (with the alias) does not show up as an embedded hyperlink to the docs, and shows up as `[Admission webhooks][k8s-doc-admission-webhooks]`. The link just shows up as plain text at the end of the rendered page (what I take to be the alias)

Since the rendered page did not link to the page on hover/click, I took it to be an error in the rendering, or a mistake.

If this is expected behaviour, I'm happy to close the PR, and apologize for the unnecessary PR!